pkg/db/model/vulnerability.go

```diff
@@ -34,22 +35,35 @@ func NewVulnerabilityModel(vulnerability db.Vulnerability) VulnerabilityModel {
 		panic(err)
 	}
 
-	proxy, err := json.Marshal(vulnerability.ProxyVulnerabilities)
+	proxy, err := json.Marshal(vulnerability.RelatedVulnerabilities)
+	if err != nil {
+		// TODO: just no
```

Comment:
I know this PR isn't set to address this, but this is becoming the default now: it doesn't get fixed and new code just copies it over with the TODO included. Since this PR is copying that behavior, either remove the TODO, or follow up with an issue to track the removal

Reply:
https://github.com/anchore/grype-db/issues/24